packages/fast-components-react-base/src/toolbar/toolbar.props.ts

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import {
+    ManagedClasses,
+    ToolbarClassNameContract,
+} from "@microsoft/fast-components-class-name-contracts-base";
+import { Orientation } from "@microsoft/fast-web-utilities";
+
+export interface ToolbarManagedClasses extends ManagedClasses<ToolbarClassNameContract> {}
+export interface ToolbarUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+export interface ToolbarHandledProps extends ToolbarManagedClasses {
+    /**
+     * The toolbar children
+     */
+    children?: React.ReactNode;
+
+    /**
+     * When true, the toolbar will immediately place focus on the appropriate focusable child when the toolbar mounts
+     */
+    enableAutoFocus?: boolean;
```

Comment:
This is not necessary, should autofocus be needed on an item in the toolbar, the user can place it on the children themselves.

Reply:
Getting rid of it in next version, will be a new PR but going over feedback so it isn't lost.